### DISCUSSION TITLE
Notifications from shoutrrr

### DISCUSSION THREAD

**Discussion by Logonoff410** [containrrr/watchtower#1378] [Q&A]
Hello!
I use the shoutrrr to receive notifications in telegram. The problem is the following - I get only the first 5 messages when starting the docker container. I no longer receive the following messages like ""Session done" Failed=0 Scanned=7 Updated=0 notify=no". How to fix it ? help plz. thx
![img-2022-09-07-12-19-31](https://user-images.githubusercontent.com/24722703/188843205-d1e06184-84e7-4025-b8e0-c4af0999d251.png)

**Comment by piksel** [comment]
Hello!

The "Session done" message is never sent as a notification, but if there were any updates you would instead get log messages about those updates.
If you really want to get such a message after each session, it can *soon* be accomplished by using a custom message template that does not discriminate against uneventful sessions 😁. The new templates are available in `:latest-dev`, but there isn't much documentation for it yet.

**Reply by Logonoff410** [reply]
Thx for the answer! Could you give an example of a templat that will work?
